lib/src/widgets/calendar/calendar_utils.dart

```diff
@@ -0,0 +1,169 @@
+library utils;
+
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+class Utils {
```

Comment:
`Utils` is really not a very descriptive name. Maybe `DateUtils` or `DateHelper` or something. `Utils` doesn't really say what it does.

Reply:
And the location of the file should be under the `util` folder as well